config/prod/s3-ingestion.yaml

```diff
@@ -0,0 +1,19 @@
+# NOTE:
+# This is the first bucket in the ingress pipeline for RECOVER project.
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.0/templates/S3/synapse-external-bucket.j2"
+stack_name: "recover-ingestion-prod"
```

Comment:
Nit: Could we for consistency, could we follow recover's pattern for naming stacks and bucket_names (see example for prod buckets): https://github.com/Sage-Bionetworks/recover/blob/main/config/prod/s3-processed-data-bucket.yaml#L4,L8

Something like:
`BucketName: "recover-ingestion"`
`stack_name: "recover-ingestion-bucket"`